src/grader/charts.js

```diff
@@ -753,7 +753,7 @@ export const SCORE_TABLE = new Map([
       description: `<p>
       <code>X-Frame-Options</code> (XFO) header set to <code>SAMEORIGIN</code> or <code>DENY</code>.
       </p>`,
-      modifier: 0,
+      modifier: 5,
       recommendation: `<p>
       Implement frame-ancestors CSP.
       </p>`,
```

Comment:
Should this recommendation also be removed, given the response in https://github.com/mdn/mdn-http-observatory/issues/270?

> observatory shouldn’t prefer one over the other

Reply:
I have removed the old recommentation.